src/test/java/io/odh/test/e2e/standard/UninstallST.java

```diff
@@ -63,56 +66,75 @@ public class UninstallST extends StandardAbstract {
         steps = {
             @Step(value = "Create uninstall configmap", expected = "ConfigMap exists"),
             @Step(value = "Wait for controllers namespace deletion", expected = "Controllers namespace is deleted"),
-            @Step(value = "Remove Operator namespace", expected = "Operator namespace is deleted"),
-            @Step(value = "Check that all related namespaces are deleted (monitoring, notebooks, controllers)", expected = "All related namespaces are deleted")
+            @Step(value = "Check that relevant resources are deleted (Subscription, InstallPlan, CSV)", expected = "All relevant resources are deleted"),
+            @Step(value = "Check that all related namespaces are deleted (monitoring, notebooks, controllers)", expected = "All related namespaces are deleted"),
+            @Step(value = "Remove Operator namespace", expected = "Operator namespace is deleted")
         }
     )
     @Test
     void testUninstallSimpleScenario() {
-        if (!ResourceManager.getKubeCmdClient().namespace(OdhConstants.OLM_OPERATOR_NAMESPACE)
-                .list("configmap").contains(DELETE_CONFIG_MAP_NAME)) {
-            ConfigMap cm = new ConfigMapBuilder()
-                    .withNewMetadata()
-                    .withName(DELETE_CONFIG_MAP_NAME)
-                    .withNamespace(OdhConstants.OLM_OPERATOR_NAMESPACE)
-                    .withAnnotations(Map.ofEntries(Map.entry(DELETE_ANNOTATION, "true")))
-                    .endMetadata()
-                    .build();
-            ResourceManager.getInstance().createResourceWithWait(cm);
-        } else {
-            Assertions.fail(String.format("The configmap '%s' is present on the cluster before the uninstall test started!", DELETE_CONFIG_MAP_NAME));
+        if (ResourceManager.getKubeCmdClient().namespace(OdhConstants.OLM_OPERATOR_NAMESPACE).list(
+                "configmap").contains(DELETE_CONFIG_MAP_NAME)) {
+            Assertions.fail(
+                    String.format("The ConfigMap '%s' is present on the cluster before the uninstall test started!",
+                            DELETE_CONFIG_MAP_NAME));
         }
 
+        ConfigMap cm = new ConfigMapBuilder()
+                .withNewMetadata()
+                .withName(DELETE_CONFIG_MAP_NAME)
+                .withNamespace(OdhConstants.OLM_OPERATOR_NAMESPACE)
+                .withLabels(Map.ofEntries(Map.entry(DELETE_ANNOTATION, "true")))
+                .endMetadata()
+                .build();
+        ResourceManager.getInstance().createResourceWithWait(cm);
+
         // Now the product should start to uninstall, let's wait a bit and check the result.
         TestUtils.waitFor(String.format("the '%s' namespace to be removed as operator is being uninstalled",
-                        OdhConstants.CONTROLLERS_NAMESPACE), 2000, 20000,
+                        OdhConstants.CONTROLLERS_NAMESPACE), 2000, 120_000,
```

Comment:
I think I'd like a constant for this, but I'm not sure about the name and scope (just for this class, or global).